conf/messages

```diff
@@ -3862,9 +3862,9 @@ mandatoryEvidenceExpired.p2=You’ll need to try again and upload a document sho
 mandatoryEvidenceExpired.label=Deleted
 mandatoryEvidenceExpired.button.text=Try Again
 
-mandatoryEvidence.scan-progress.title=Wait a few seconds and then select 'continue'
+mandatoryEvidence.scan-progress.title=Wait a few seconds and then select ''continue''
```

Comment:
these should be curly quotes like the ones on this page: https://smartquotesforsmartpeople.com/